http/org_service.go

```diff
@@ -647,60 +645,42 @@ func (s *OrganizationService) FindOrganizations(ctx context.Context, filter infl
 	span, _ := tracing.StartSpanFromContext(ctx)
 	defer span.Finish()
 
+	var params [][2]string
 	if filter.Name != nil {
 		span.LogKV("org", *filter.Name)
+		params = append(params, [2]string{"org", *filter.Name})
 	}
 	if filter.ID != nil {
 		span.LogKV("org-id", *filter.ID)
+		params = append(params, [2]string{"orgID", filter.ID.String()})
 	}
 	for _, o := range opt {
 		if o.Offset != 0 {
 			span.LogKV("offset", o.Offset)
+			params = append(params, [2]string{"offset", strconv.Itoa(o.Offset)})
```

Comment:
options decoding logic is all located in `decodeFindOptions` in `http/paging.go`. I'd prefer not to append these shared params in every http service if possible.

Reply:
Honestly, the fact that this is looking at options here just for tracing feels like it could be DRY'd, but it doesn't seem like this is uniformly done in other services.

Reply:
I'll create a similar method that'll return the params 👍